Archipelago.MultiClient.Net/Enums/Permissions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Archipelago.MultiClient.Net.Enums
+{
+	[JsonConverter(typeof(StringEnumConverter))]
+	[Flags]
+	public enum Permissions

```

Comment:
Might make more sense to use singular here?

Reply:
Maybe i don't work with flags that often so i googled up recommended naming and micrsoft said singular for enums but plural for flags